python/ray/dag/compiled_dag_node.py

```diff
@@ -464,6 +482,24 @@ def __init__(
         # this DAG, if any.
         self._nccl_group_id: Optional[str] = None
 
+        # Creates the driver actor.
+        #
+        # To support the driver as a reader, the output writer needs to be able to
+        # invoke remote functions on the driver. This is necessary so that the output
+        # writer can create a reader ref on the driver node, and later potentially
+        # create a larger reader ref on the driver node if the channel backing store
+        # needs to be resized. However, remote functions cannot be invoked on the
+        # driver.
+        #
+        # An accelerated DAG creates an instance of this class when the DAG is
+        # initialized. This class has an empty implementation, though it serves as a way
+        # for the output writer to invoke remote functions on the driver node.
```

Comment:
Can you shorten this comment to just explain the part that's relevant to this code? It's a bit verbose right now and repeats most of what is written in the docstring.